internal/pkg/vm/actor/builtin/miner/miner.go

```diff
@@ -299,55 +237,201 @@ var minerExports = exec.Exports{
+	ErrSectorIDInUse = 35
+	// ErrStoragemarketCallFailed indicates the call to commit the deal failed.
+	ErrStoragemarketCallFailed = 36
+	// ErrCallerUnauthorized signals an unauthorized caller.
+	ErrCallerUnauthorized = 37
+	// ErrInsufficientPledge signals insufficient pledge for what you are trying to do.
+	ErrInsufficientPledge = 38
+	// ErrInvalidPoSt signals that the passed in PoSt was invalid.
+	ErrInvalidPoSt = 39
+	// ErrAskNotFound indicates that no ask was found with the given ID.
+	ErrAskNotFound = 40
+	// ErrInvalidSealProof signals that the passed in seal proof was invalid.
+	ErrInvalidSealProof = 41
+	// ErrGetProofsModeFailed indicates the call to get the proofs mode failed.
+	ErrGetProofsModeFailed = 42
+	// ErrInsufficientCollateral indicates that the miner does not have sufficient collateral to commit additional sectors.
+	ErrInsufficientCollateral = 43
+	// ErrMinerAlreadySlashed indicates that an attempt has been made to slash an already slashed miner
+	ErrMinerAlreadySlashed = 44
+	// ErrMinerNotSlashable indicates that an attempt has been made to slash a miner that does not meet the criteria for slashing.
+	ErrMinerNotSlashable = 45
+	// ErrInvalidPieceInclusionProof indicates that the piece inclusion proof was
+	// malformed or did not succesfully verify.
+	ErrInvalidPieceInclusionProof = 46
+)
+
+// Errors map error codes to revert errors this actor may return.
+var Errors = map[uint8]error{
+	ErrInvalidSector:              errors.NewCodedRevertErrorf(ErrInvalidSector, "sectorID out of range"),
+	ErrSectorIDInUse:              errors.NewCodedRevertErrorf(ErrSectorIDInUse, "sector already committed at this ID"),
+	ErrStoragemarketCallFailed:    errors.NewCodedRevertErrorf(ErrStoragemarketCallFailed, "call to StorageMarket failed"),
+	ErrCallerUnauthorized:         errors.NewCodedRevertErrorf(ErrCallerUnauthorized, "not authorized to call the method"),
+	ErrInsufficientPledge:         errors.NewCodedRevertErrorf(ErrInsufficientPledge, "not enough pledged"),
+	ErrInvalidPoSt:                errors.NewCodedRevertErrorf(ErrInvalidPoSt, "PoSt proof did not validate"),
+	ErrAskNotFound:                errors.NewCodedRevertErrorf(ErrAskNotFound, "no ask was found"),
+	ErrInvalidSealProof:           errors.NewCodedRevertErrorf(ErrInvalidSealProof, "seal proof was invalid"),
+	ErrGetProofsModeFailed:        errors.NewCodedRevertErrorf(ErrGetProofsModeFailed, "failed to get proofs mode"),
+	ErrInsufficientCollateral:     errors.NewCodedRevertErrorf(ErrInsufficientCollateral, "insufficient collateral"),
+	ErrInvalidPieceInclusionProof: errors.NewCodedRevertErrorf(ErrInvalidPieceInclusionProof, "piece inclusion proof did not validate"),
+}
+
+const (
+	PoStStateNoStorage = iota
+	PoStStateWithinProvingPeriod
+	PoStStateAfterProvingPeriod
+	PoStStateUnrecoverable
+)
+
 // AddAsk adds an ask to this miners ask list
-func (ma *Actor) AddAsk(ctx exec.VMContext, price types.AttoFIL, expiry *big.Int) (*big.Int, uint8,
```

Comment:
Similarly, I don't object to this rename on its own, but it would be far better to have happened in a different PR. This also means that the functional change can't be reverted independently to these renames etc etc.